src/Adapters/EventSource.Shared/EventSource.Shared/Implementation/EventSourceListenerEventSource.cs

```diff
@@ -19,6 +19,8 @@ internal sealed class EventSourceListenerEventSource : EventSource
         public static readonly EventSourceListenerEventSource Log = new EventSourceListenerEventSource();
 
         private const int NoEventSourcesConfiguredEventId = 1;
+        private const int FailedToEnableProvidersEventId = 2;
+        private const int ModuleInitializationFailedEventId = 3;
 
         private EventSourceListenerEventSource()
```

Comment:
Cannot find event for "need to run under priviledged account". When you'll create it - mark it with "UserActionable" keyword and make it Error. So it will be send to portal and customer will see it